weiluo/Community_Contribution.Rmd

```diff
@@ -0,0 +1,125 @@
+
+#Using PostgreSQL Databse in R with MacOS Environment
+wl2824 Wei Luo
```

Comment:
would probably be preferable to remove your CUID and have this line just say "Wei Luo"